appscale/tools/templates/AppScalefile

```diff
@@ -0,0 +1,162 @@
+   - [vm6_private_IP | node-6]
+  zookeeper:  # Zookeeper needs a majority of peers online (hence 3)
+   - [vm7_private_IP | node-7]
+   - [vm8_private_IP | node-8]
+   - [vm9_private_IP | node-9]
+login : [vm1_public_IP | Domain Name]
+
+
+
+#################################################
+########## Public Cloud configuration ###########
+#################################################
+
+############ Amazon Web Services EC2 ############
+# To deploy on AWS EC2, uncomment/fill out the following parameters:
+# infrastructure : 'ec2'
+
+# AppScale-ready AWS AMI.
+# machine : 'ami-XXXXXX’
+
+# AWS credentials.
+# EC2_ACCESS_KEY: 'YYYY'
+# EC2_SECRET_KEY: 'ZZZZ'
+
+# Instance type of your choice. Default is m3.medium.
+# instance_type : 'm3.medium'
+
+# The availability zone that should be used for all virtual machines in your
+# AppScale deployment. If using persistent disks in your AppScale deployment,
+# they should be in the same zone that you specify here.
+# zone : 'us-east-1b'
+
+# The unique security and network group name that will be used for all machines
+# in this AppScale deployment.
+# By default, the group is auto-generated.
+# group : 'appscalegroup'
+
+# The Elastic IP that will be used as (part of) the application frontend URL.
+# static_ip: '1.2.3.4'
+
+
+############ Google Compute Engine ############
+# To deploy on GCE, uncomment/fill out the following parameters:
+# infrastructure : 'gce'
+
+# GCE Project ID.
+# project: 'project-id-here'
+
+# AppScale-ready VM Image.
+# machine : ‘image-name-here’
+
+# GCE credentials.
+# client_secrets: '~/client_secrets.json'
+
+# Instance type of your choice. Default is n1-standard-1.
+# instance_type : 'n1-standard-1'
+
+# The availability zone that should be used for all virtual machines in your
+# AppScale deployment. If using persistent disks in your AppScale deployment,
+# they should be in the same zone that you specify here.
+# zone : 'us-central1-a'
+
+# The unique network and firewall rule name for the machines belonging to
+# this deployment.
+# By default, the group is auto-generated.
+# group : 'appscalegroup'
+
+
+############ Microsoft Azure ############
+# To deploy on Azure, uncomment/fill out the following parameters:
+# infrastructure : ‘azure’
+
+# AppScale-ready VM image URL.
+# machine : ‘URL-to-image-here’
+
+# Azure credentials.
+# azure_subscription_id : ‘JJJJ’
+# azure_app_id : ‘KKKK’
+# azure_tenant_id : ‘XXXX’
+# azure_app_secret_key : ‘YYYY’
+
+# Instance type of your choice. Default is Standard_A3.
+# instance_type : ‘Standard_A3’
+
+# The availability zone that should be used for all virtual machines in your
+# AppScale deployment. If using persistent disks in your AppScale deployment,
+# they should be in the same zone that you specify here.
+# zone : ‘West-US’
+
+# The unique resource group that Azure resources will be created under for this
+# deployment.
+# By default, the group is auto-generated.
+# group : 'appscalegroup'
```

Comment:
The resource group parameter to use for Azure is named as 'azure_resource_group'. This 'group' is the common security group name used for the rest of the cloud platforms.